lib/output_helper.rb

```diff
@@ -92,15 +92,16 @@ def self.csv_line(vrp, service, cluster_index, two_stages = false)
 
   class Result
     def self.generate_geometry(solution)
-      return nil unless solution.to_h[:result]
+      return nil unless solution&.key?(:result)
```

Comment:
What if the hash has the key `:result` but it is `nil`? 

I think your first implementation `solution && solution[:result]` was better.

or you can be more expressive and do `return nil if solution.nil? || solution[:result].nil? || solution[:result].empty?` because this reads literally the conditions we care.

Reply:
I will go with your last suggestion.